ports/geckolib/gecko_bindings/bindings.rs

```diff
@@ -251,7 +253,7 @@ extern "C" {
                                           classList: *mut *mut *mut nsIAtom)
      -> u32;
     pub fn Gecko_GetServoDeclarationBlock(element: *mut RawGeckoElement)
-     -> *mut ServoDeclarationBlock;
+     -> ServoDeclarationBlockBorrowed;
```

Comment:
I'm surprised this compiles? This is a dangling lifetime -- there's no input lifetime to feed into it.